storage-sample.go

```diff
@@ -53,30 +49,38 @@ func fatalf(service *storage.Service, errorMessage string, args ...interface{})
 
 func restoreOriginalState(service *storage.Service) bool {
 	// Delete an object from a bucket.
-	if err := service.Objects.Delete(bucketName, objectName).Do(); err != nil {
+	if err := service.Objects.Delete(*bucketName, objectName).Do(); err != nil {
 		// If the object exists but wasn't deleted, the bucket deletion will also fail.
 		fmt.Printf("Could not delete object during cleanup: %v\n\n", err)
 	} else {
-		fmt.Printf("Successfully deleted %s/%s during cleanup.\n\n", bucketName, objectName)
+		fmt.Printf("Successfully deleted %s/%s during cleanup.\n\n", *bucketName, objectName)
 	}
 
 	// Delete a bucket in the project
-	if err := service.Buckets.Delete(bucketName).Do(); err != nil {
+	if err := service.Buckets.Delete(*bucketName).Do(); err != nil {
 		fmt.Printf("Could not delete bucket during cleanup: %v\n\n", err)
 		fmt.Println("WARNING: Final cleanup attempt failed. Original state could not be restored.\n")
 		return false
 	}
 
-	fmt.Printf("Successfully deleted bucket %s during cleanup.\n\n", bucketName)
+	fmt.Printf("Successfully deleted bucket %s during cleanup.\n\n", *bucketName)
 	return true
 }
 
 func main() {
 	flag.Parse()
-	if *jsonFile != "" {
-		os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", *jsonFile)
+	if *bucketName == "" {
+		log.Fatalf("Bucket argument is required. See --help.")
+	}
+	if *projectID == "" {
+		log.Fatalf("Project argument is required. See --help.")
 	}
+    if *fileName == "" {
+        log.Fatalf("File argument is required. See --help.")
+    }
 
```

Comment:
Note that `gofmt` should always be run on the code before pushing... so you never have to worry about indentation. I have `goimports` wired into my text editor so that on every single "save" of the file, it handles all formatting and fixes all my imports for me, so I never even think about it.  :-)